### PR TITLE
[NUI][AT-SPI] Highlight bindings

### PR DESCRIPTION
**Pull request by grzelewski**
Highlight bindings for accessibility.

### Description of Change ###
Added highlight bindings for accessibility.

### API Changes ###

 - ACR: NONE

### CODE REVIEW

File: src/Tizen.NUI/src/public/Accessibility/Accessibility.cs [hunk 1]
```diff
@@ -108,6 +108,46 @@ public void PauseResume(bool pause)
...
+        }
+
+        /// <summary>
+        ///  Set view that will be used to highlight widget.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetHighlightFrameView(View view) {
+            view.SetAsHighlightView();
+        }
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/Accessibility/Accessibility.cs:130]: Please check build warning message.

File: src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs [hunk 2]
```diff
@@ -237,6 +237,12 @@ public bool GrabAccessibilityHighlight()
             return result;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal void SetAsHighlightView() {
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs:241]: Please refer code.
```csharp
SetAsHighlightView()
{
}
```

File: src/Tizen.NUI/src/public/Accessibility/Accessibility.cs [hunk 3]
```diff
@@ -108,6 +108,49 @@ public void PauseResume(bool pause)
...
+
+        /// <summary>
+        ///  Get View that is used to highlight widget.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View GetHighlightFrameView() {
+            var ptr = Interop.ControlDevel.DaliAccessibilityAccessibleGetHighlightActor();
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return this.GetInstanceSafely<View>(ptr);
```
Comment (grzelewski) [inline src/Tizen.NUI/src/public/Accessibility/Accessibility.cs:120]: It seems that is some cases this line is not so secure and create crash.

File: src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs [hunk 4]
```diff
@@ -203,6 +203,18 @@ internal static partial class ControlDevel
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_Bridge_Remove_Popup")]
             public static extern void DaliAccessibilityBridgeRemovePopup(global::System.Runtime.InteropServices.HandleRef arg1);
 
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_Accessible_GetCurrentlyHighlightedActor")]
+            public static extern global::System.IntPtr DaliAccessibilityAccessibleGetCurrentlyHighlightedActor();
+
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_Accessible_GetHighlightActor")]
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs:211]: @grzelewski 
Hello
now I put the \*\***NO MERGE**\*\* tag on thie PR.
you can remove \*\***NO MERGE**\*\* tag for yourself when this PR is fixed and ready to be merged.
thank you.

File: src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs (hunk 2 above)
Comment (grzelewski) [reply src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs]: Done.

File: src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs [hunk 5]
```diff
@@ -244,6 +244,9 @@ public bool GrabAccessibilityHighlight()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public void AppendAccessibilityRelation(View second, RelationType relation)
         {
+            if (second is null)
+                throw new ArgumentNullException(nameof(second));
```
[the remark this answers is not in the document]
Comment (grzelewski) [reply src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs:248]: Done.

File: src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs (hunk 4 above)
Comment (grzelewski) [reply src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs:211]: Done.

**Review by Seoyeon2Kim — COMMENTED** [review]
I'll have a look at this PR again after dali integration is done.
As we know, dali-csharp-binder patch should be applied at first.

**Review by Seoyeon2Kim — APPROVED** [review]
The binding patch below was merged and applied to the latest platform binary.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/255306/
Approved this PR :)

**Pull request merged**